### DISCUSSION TITLE
Add list of photoprism instances

### DISCUSSION THREAD

**Discussion by stemy2** [photoprism/photoprism#2905] [General]
PLEASE ONLY PROCEED IF YOU ARE ABSOLUTELY SURE THAT THIS IS NOT A TECHNICAL SUPPORT INCIDENT AND/OR A PROBLEM WITH OTHER SOFTWARE YOU ARE USING. PLEASE ALSO CHECK OUR PUBLIC ROADMAP AND TRY TO FIND EXISTING FEATURE REQUESTS FIRST:

- <https://link.photoprism.app/roadmap>
- <https://github.com/photoprism/photoprism/issues>
 
VISIT <https://photoprism.app/kb/getting-support> TO LEARN MORE ABOUT OUR SUPPORT OPTIONS. THANK YOU FOR YOUR CAREFUL CONSIDERATION!

===============================================================================

**Describe what problem this solves and why this would be valuable to many users**

Self-hosting can be difficult for non avanced users, photoprism could be more accessible if you could provide an instances list.

**Describe the solution you'd like**

The instances list could be an opt-in.

**Describe alternatives you've considered**

**Additional context**

**Comment by graciousgrey** [comment]
What exactly do you mean with instances list?

**Comment by stemy2** [comment]
An list of servers where photoprism is installed and where anybody can signup.
For example, the pixelfed instances list: https://pixelfed.org/servers

**Comment by lastzero** [comment]
There's no community servers as our software is primarily for managing all your pictures in a privacy friendly way. If you were to upload all your private pictures to a community server, that wouldn't service the purpose. However, there is a hosted photoprism service if you don't want to self host it: https://docs.photoprism.app/getting-started/cloud/pikapods/
